src/cpp/src/continuous_batching_impl.cpp

```diff
@@ -403,11 +406,13 @@ ContinuousBatchingPipeline::ContinuousBatchingImpl::generate(const std::vector<o
         try {
             const auto infer_start = std::chrono::steady_clock::now();
             step();
+            // The very first prefill step we don't generate new tokens, but still inference took place,
```

Comment:
whether we generate new tokens or not, depends on max_batch_size.
If max_batch_size < prompt_len, then prompt is split into chunks and processed one by one. 

So, comment can be adjust to reflect actual logic, because says that "first inference does not generate tokens" is not always correct.

Reply:
adjusted comment accordingly